build_tools/cmake/build_riscv.sh

```diff
@@ -45,6 +45,8 @@ args=(
   -DRISCV_COMPILER_FLAGS="${RISCV_COMPILER_FLAGS}"
   -DIREE_ENABLE_ASSERTIONS=ON
   -DIREE_BUILD_COMPILER=OFF
+  # CPU info doesn't work on RISCV
+  -DIREE_ENABLE_CPUINFO=OFF
```

Comment:
It can be optional. We don't use cpuinfo in RV64 for any real function. The warning log is annoying but not hurtful.

Reply:
I don't really follow. Do you think I shouldn't disable this? The warning isn't terrible, but I don't think disabling hurts either

Reply:
Agree it disabling the warning doesn't hurt, although it could be argued for a different clean up PR. :)

Maybe we should put the flag under the condition of L54-56 since that's where the cpuinfo is actually used?

Reply:
So it does work on rv32? The warning just says that CPUInfo doesn't work on risc arch. The difference between this change and the others you suggested is that this one I understand what it's doing :-)

Reply:
rv32-baremetal runtime artifacts are not built with cpuinfo library at all. You should not see any cpuinfo warning message in rv32's CI logs

Reply:
Oh sure, but disabling it is also fine I think? I can revert this line if you'd prefer. Maybe we should just change the default for `IREE_ENABLE_CPUINFO` based on arch also

Reply:
The default should be ON since most of the target platforms can use cpuinfo without a problem. Disabling cpuinfo for all RISCV build scripts should be a bit overkill but it's not breaking anything either. I am okay of keeping it where it is now but would prefer it gets moved down.

Reply:
The default is `ON`, but we can have a different default per arch and therefore make it default off for RISCV. To me, it makes more sense to not have it in the condition because that would sort of imply that it worked for rv32 (whereas it sounds like it doesn't get build there for some other reason).

Reply:
Understood. Let's keep the assignment as is.